qiskit_machine_learning/neural_networks/two_layer_qnn.py

```diff
@@ -115,9 +115,10 @@ def __init__(
         operator = ~StateFn(self.observable) @ StateFn(self._circuit)
 
         super().__init__(
-            operator,
-            input_params,
-            weight_params,
+            operator=operator,
+            input_params=input_params,
+            weight_params=weight_params,
+            exp_val=exp_val,
```

Comment:
I note that `exp_val` is not in the args docstring for the constructor of this class being altered (i.e. two_layer_qnn), nor is 'quantum_instance` it seems,

Reply:
Interesting. I added them, thanks for spotting.